client/me/privacy/main.jsx

```diff
@@ -0,0 +1,41 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import { localize } from 'i18n-calypso';
+import { flowRight as compose } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { protectForm } from 'lib/protect-form';
+import formBase from 'me/form-base';
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+
+// NOTE: disabling it since we need to use the `formBase` mixin
+// eslint-disable-next-line react/prefer-es6-class
+const Privacy = createReactClass( {
+	displayName: 'Privacy',
+
+	mixins: [ formBase ],
```

Comment:
What does formBase do? We can try porting functionality to HoC if we really need it.

Reply:
Ah so if we look at the readme, there are a few options here. There's both a [HoC](https://github.com/Automattic/wp-calypso/tree/aafc155b7c3d67d054b29fc32eccc07655f0a2e2/client/lib/protect-form#usage-hoc) available in addition to a [declarative component](https://github.com/Automattic/wp-calypso/tree/aafc155b7c3d67d054b29fc32eccc07655f0a2e2/client/lib/protect-form#usage-declarative).

Let's remove this for now and add it back when we actually have a form or toggle to protect changes for.